docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+    "Probably not necessary to use `use_monitor=False`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def run_sscan():\n",
+    "    yield from bps.mv(scan_button, 1)\n",
+    "    print(\"Started scan ...\")\n",
+    "    yield from bps.sleep(0.1)  # wait for scan to start\n",
+    "    while scan_button.get(use_monitor=False) != 0:\n",
+    "        yield from bps.sleep(0.1)\n",
+    "    print(\"Scan done.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Execute the `run_sscan()` plan using the bluesky RunEngine `RE`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Started scan ...\n",
+      "Scan done.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get the data from the `sscan` record, we use `ophyd.EpicsSignalRO` to connect with the arrays.  We'll do that in a later section of this notebook.\n",
+    "\n",
+    "### 1.B. wait with a Status object\n",
+    "\n",
+    "Let's eliminate that polling loop.  This will make the plan respond more quickly\n",
+    "when the button changes value.  The `ophyd.status.Status` class (runs in the\n",
+    "background) reports when the scan has finished or times out.  We define an inner\n",
+    "`watch_the_button()` function to detect when the button goes from `1` to `0`\n",
+    "(the scan stops).  We setup a subscription (*after* we start the move) to\n",
+    "monitor the button value and respond immediately.  \n",
+    "\n",
+    "We wait for the status object to finish with `st.wait()`.  In a bluesky plan,\n",
+    "we'll need to use `apstools.plans.run_blocking_function` with `st.wait()` since\n",
+    "it is a blocking function.\n",
+    "\n",
+    "Finally, we cancel the monitor of the button."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from apstools.plans import run_blocking_function\n",
+    "from ophyd.status import Status\n",
+    "\n",
+    "def run_sscan():\n",
+    "    st = Status(timeout=20)\n",
+    "\n",
+    "    def watch_the_button(old_value, value, **kwargs):\n",
+    "        if old_value == 1 and value == 0:\n",
+    "            st.set_finished()\n",
+    "            scan_button.clear_sub(watch_the_button)\n",
+    "\n",
+    "    yield from bps.mv(scan_button, 1)\n",
+    "    scan_button.subscribe(watch_the_button)\n",
+    "    yield from run_blocking_function(st.wait)"
```

Comment:
The rest of the code is not clear, I kind of understand, but not quite, so if I were someone new to bluesky I would probably ask very basic questions like:

- Why do we need both `scan_button.subscribe(watch_the_button)` and `yield from run_blocking_function(st.wait)`? what do they do?
- Why does `scan_button.subscribe(watch_the_button)` does not need a `yield from`?
- there are no argument passed to `watch_execute_scan` so I guess it works like a signal/slot - I read the doc for subscribe but it is a not obvious what get passed here. Need more explanations

Reply:
Added a comment `# Once the scan finishes, scan1.EXSC changes from 1 to 0.` 

Added a `<details>` section.

> In a bluesky plan, we'll need to use `apstools.plans.run_blocking_function` with `st.wait()` since it is a blocking function.